otx/algorithms/classification/configs/efficientnet_b0_cls_incr/template_selfsl.yaml

```diff
@@ -0,0 +1,62 @@
+# Description.
```

Comment:
`train_type` option could be specified via otx CLI, so we just don't need one more copy of template with a different `train_type`.

Overall, training paradigm choice should be done via the `train_type` parameter (supervised, semu-sl, self-sl). Also, we have a subdivision into multiclass, multilabel and hierarchical methods. It's done in runtime and based on the input label schema (and it can't be specified via template in advance). This scheme is not perfect, but at least for now we try to keep only one template per classification model and also this is a requirement from the geti's architecture side.

Reply:
I couldn't find how to specify the `train_type` in the docs. Would you mind showing me how to do it?

Reply:
Thank you :)